app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,29 @@
+class SessionsController < Devise::SessionsController
+  def new
+    super
+  end
+
+  def create
+    if(current_company == nil)
+      flash[:danger] = 'Invalid Company!'
+      return redirect_to new_employee_session_path
+    end
+    employee = Employee.find_by(email: params[:employee][:email], company_id: current_company.id)
+    if employee == nil || !employee.valid_password?(params[:employee][:password])
+      flash[:danger] = "Invalid password or email!"
+      return redirect_to new_employee_session_path
+    end
+    if current_company.subdomain == employee.company.subdomain
+      clean_up_passwords(employee)
+      sign_in(resource_name, employee)
+      yield employee if block_given?
+      if !flash.key?('alert')
```

Comment:
flash[:alert].blank?

Reply:
then success message in green will be displayed else there must be error

Reply:
use flash[:alert].blank? instead of flash.key?('alert')